library/systemd/src/lib/yast2/systemd/service.rb

```diff
@@ -0,0 +1,216 @@
+require "yast2/systemd/unit"
+
+module Yast2
+  module Systemd
+    # Represent a missed service
+    class ServiceNotFound < StandardError
+      def initialize(service_name)
+        super "Service unit '#{service_name}' not found"
+      end
+    end
+
+    # Systemd.service unit control API
+    #
+    # @example How to use it in other yast libraries
+    #    require 'yast'
+    #    require 'yast2/systemd/service'
+    #
+    #    ## Get a service unit by its name
+    #    ## If the service unit can't be found, you'll get nil
+    #
+    #    service = Yast::Systemd::Service.find('sshd') # service unit object
```

Comment:
It should be `Yast2::Systemd::Service`. Please, check other occurrences in this documentation.